@rlewczuk 
There seems to be a strange clang tidy error:
https://github.com/tenstorrent/tt-metal/actions/runs/25343657223/job/74307404024#step:18:34
```
/work/ttnn/cpp/ttnn/operations/core/compute_kernel/compute_kernel_config.hpp:11:10: error: 'umd/device/types/arch.hpp' file not found [clang-diagnostic-error]
   11 | #include <umd/device/types/arch.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
19707 warnings and 1 error generated.
```

I requeued again. But if this fails again with the same error, you should double check what is going wrong.

--- a/ttnn/cpp/ttnn/operations/transformer/CMakeLists.txt
+++ b/ttnn/cpp/ttnn/operations/transformer/CMakeLists.txt
@@ -24,6 +24,13 @@ target_sources(
         TYPE HEADERS
         BASE_DIRS ${FixmeOpAPIDir}
         FILES
+            sdpa_config.hpp
+            attention_softmax/attention_softmax.hpp
+            concatenate_heads/concatenate_heads.hpp
+            sdpa/sdpa.hpp
+            sdpa_decode/sdpa_decode.hpp
+            sdpa_windowed/sdpa_windowed.hpp
+            split_query_key_value_and_split_heads/split_query_key_value_and_split_heads.hpp
         FILE_SET kernels
         TYPE HEADERS
         BASE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}
@@ -54,12 +61,7 @@ target_sources(
 )
 
 target_include_directories(ttnn_op_transformer PRIVATE ${FixmeOpIncDirs})
-target_link_libraries(
-    ttnn_op_transformer
-    PRIVATE
-        TT::Metalium
-        TTNN::Core
-)
+target_link_libraries(ttnn_op_transformer PRIVATE TT::Metalium PUBLIC TTNN::Core)
 
 install(
     TARGETS
@@ -71,3 +73,4 @@ install(
 )
 
 install(TARGETS ttnn_op_transformer LIBRARY COMPONENT tar)
+install(TARGETS ttnn_op_transformer FILE_SET api COMPONENT ttnn-dev LIBRARY COMPONENT tar)

